pulp_smash/tests/rpm/api_v2/test_force_full.py

```diff
@@ -0,0 +1,200 @@
+# coding=utf-8
+"""Test re-publish repository after unassociating content.
```

Comment:
This module-wide docstring doesn't seem to match [Pulp #1158](https://pulp.plan.io/issues/1158) or the test case below. Am I missing something?